### DISCUSSION TITLE
What's correct lore-based regional settings?

### DISCUSSION THREAD

**Discussion by Dushess** [Aelto/W3_RandomEncounters_Tweaks#187] [Q&A] [answered]
Just everyone can be encountered everywheer, not good I presume.

**Comment by Aelto** [comment]
The default settings. If you apply it you'll see the Toussaint creatures are restricted to Toussaint only, the Skellige creatures are restricted to Skellige only, and almost all monsters are removed from white orchard.

**Comment by Dushess** [comment, marked as answer]
I mean what I should select for a location, Didn't played it well yet, so I have no idea about correct pattern.

**Reply by Dushess** [reply]
Ooh, defaults is actually working, so I'll use it.

**Reply by Aelto** [reply]
You want all creatures to be able to spawn everywhere? There is a `everywhere` preset you can pick in the region constraints menu
